src/import-data/protected-data-utils.ts

```diff
@@ -57,13 +57,16 @@ export const isProtectedSource = (importRequest: ImportRequest): boolean => {
   }
 };
 
-// This method identifies whether an import source is an AnVIL Explorer import
-export const isAnvilImport = (url: URL): boolean => {
+// This method identifies an import source. Currently it only identifies AnVIL Explorer.
+export const getImportSource = (url: URL): string => {
   const anvilSources = [
     'service.prod.anvil.gi.ucsc.edu',
     'edu-ucsc-gi-platform-anvil-prod-storage-anvilprod.us-east-1',
   ];
-  return anvilSources.some((path) => url.href.includes(path));
+  if (anvilSources.some((path) => url.href.includes(path))) {
+    return 'anvil';
```

Comment:
Might be good to define an enum Type here to cover your expected values, then return that instead of a string

Reply:
I figured a single type wasn't worth it but that's fair.

Reply:
I don't actually know the most idiomatic way to do this, but maybe it's as simple as what's described here: https://www.typescriptlang.org/docs/handbook/enums.html#string-enums

I vaguely recall other places I've seen where you've got a string subclass whose only valid values are a specific set.

Reply:
Generally, we've preferred literal union types over enums in Terra UI for reasons described in the linked article.

A literal union type can be declared like:
```ts
type Source = 'someValue' | 'anotherValue' | 'yetAnotherValue';
```

- https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#literal-types
- https://www.bam.tech/en/article/should-you-use-enums-or-union-types-in-typescript

Reply:
Thank you, i looked and couldn't find enums in the codebase and was starting to sweat

Reply:
Added importSource `type` as the return type of getImportSource